.copywrite.hcl

```diff
@@ -0,0 +1,27 @@
+project {
+  # Leave blank if you don't wish to license the project
+  # Default: "MPL-2.0"
+  license = "MPL-2.0"
+
+  # (OPTIONAL) Represents the year that the project initially began
+  # Default: <the year the repo was first created>
+  copyright_year = 2013
+
+  # (OPTIONAL) A list of globs that should not have copyright or license headers .
+  # Supports doublestar glob patterns for more flexibility in defining which
+  # files or folders should be ignored
+  # Default: []
+  header_ignore = [
+    "*.hcl2spec.go",
+    "hcl2template/testdata/**",
+    "test/**",
+    "**/test-fixtures/**",
+    "examples/**",
+    "hcl2template/fixtures/**",
+    "website/**",
```

Comment:
@lbajolet-hashicorp do you mind adding some comments on any of these that are notable? I believe I recall the `hcl2spec` and `hcl2template` ones being part of autogenerated code. Just thinking for the future in case any questions come up about an entry 🙂

Reply:
Done, I've left two comments on the non-test data here